package.json

```
@@ -0,0 +1,104 @@
+{
+  "name": "network-dashboard",
+  "description": "Aragon Network Dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "license": "AGPL-3.0-or-later",
+  "author": "Aragon Association <legal@aragon.org>",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/aragon/network-dashboard.git"
+  },
+  "bugs": {
+    "url": "https://github.com/aragon/network-dashboard/issues"
+  },
+  "homepage": "https://github.com/aragon/network-dashboard#readme",
+  "keywords": [
+    "aragon",
+    "network",
+    "ethereum",
+    "dapp",
+    "blockchain",
+    "startups"
+  ],
+  "engines": {
+    "node": ">= 12.0.0"
+  },
+  "bundlewatch": {
+    "files": [
+      {
+        "path": "public/*.js",
+        "maxSize": "1600kb",
+        "compression": "gzip"
+      }
+    ]
+  },
+  "dependencies": {
+    "@aragon/ui": "^1.5.0",
+    "core-js": "^3.3.2",
+    "dotenv": "^8.2.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-spring": "^8.0.1",
+    "rimraf": "^3.0.2",
+    "styled-components": "^5.1.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.2.2",
+    "babel-plugin-styled-components": "^1.11.1",
+    "bundlewatch": "^0.2.7",
+    "cross-env": "^7.0.2",
+    "eslint": "^7.6.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-config-standard-react": "^9.2.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jest": "^23.20.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^4.0.8",
+    "eslint-plugin-standard": "^4.0.1",
+    "husky": "^4.2.5",
+    "jest": "^26.2.2",
+    "lint-staged": "^10.2.11",
+    "parcel-bundler": "^1.12.4",
+    "parcel-plugin-bundle-visualiser": "^1.2.0",
+    "prettier": "^2.0.5"
+  },
+  "scripts": {
+    "bundlewatch": "bundlewatch",
+    "start": "rimraf ./public && npm run sync-assets && parcel serve src/index.html --port 3000 --out-dir ./public",
+    "start:mainnet": "cross-env CHAIN_ID=1 npm start ",
+    "start:rinkeby": "cross-env CHAIN_ID=4 npm start",
+    "build": "rimraf ./public && ./scripts/build.sh",
+    "build:mainnet": "cross-env CHAIN_ID=1 npm run build",
+    "build:rinkeby": "cross-env CHAIN_ID=4 npm run build",
+    "lint": "eslint ./src",
+    "test": "npm run lint && npm run jest",
+    "jest": "jest --passWithNoTests",
+    "sync-assets": "copy-aragon-ui-assets ./public"
+  },
+  "browserslist": {
+    "development": ">2%, last 1 edge versions, not ie > 0, not op_mini all",
+    "production": ">2%, last 1 edge versions, not ie > 0, not op_mini all"
+  },
```

I think this should work:

```suggestion
  "browserslist": [
    ">2%",
    "last 1 edge versions",
    "not ie > 0",
    "not op_mini all",
  ],
```